packages/sanes-chrome-extension/src/extension/background/model/persona/index.ts

```diff
@@ -27,6 +27,7 @@ export interface ProcessedTx {
   readonly amount: Amount;
   readonly memo?: string;
   readonly time: string;
+  readonly blockExplorerUrl: string | null;
```

Comment:
This is the direct link to the transaction, right?

Reply:
yep, link to tx in etherscan or other block explorer.